spec/abstractops.html

```diff
@@ -294,6 +294,49 @@ <h1>RoundNumberToIncrement ( _x_, _increment_, _roundingMode_ )</h1>
     </emu-alg>
   </emu-clause>
 
+  <emu-clause id="sec-temporal-roundtime" aoid="RoundTime">
+    <h1>RoundTime ( _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanosecond_, _increment_, _unit_, _roundingMode_ )</h1>
```

Comment:
Should this be in `time.html`?

Reply:
I put it here because it was also used in DateTime, but it could go there as well.